experimental/alp/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+get_property(dialect_libs GLOBAL PROPERTY MLIR_DIALECT_LIBS)
+get_property(conversion_libs GLOBAL PROPERTY MLIR_CONVERSION_LIBS)
+
+set(LLVM_LINK_COMPONENTS
+  Core
+  Support
+  nativecodegen
```

Comment:
I think native for you is AArch64 ?
For us it is still X86.
I don't think it is a problem for now but we should also think about setting things up so we can cross-compile and look at assembly at some ponit.
I am interested in seeing arm assembly even if I cannot run it myself.